clients/src/test/java/org/apache/kafka/common/utils/TimeTest.java

```diff
@@ -80,4 +82,11 @@ public void testWaitObjectConditionSatisfied() throws InterruptedException {
         assertTrue(time.milliseconds() < deadlineMs);
         assertNull(caughtException.get());
     }
+
+    @Test
+    public void testOnlyOneSystemTime() {
```

Comment:
We don't need to test Java's variable finality, I think it is safe to assume this will work.
In other words, this test would pass regardless of the implementation of the Time interface, so it can't possibly detect any flaw in the code.

Reply:
Thanks for the reminder, I will delete it